.github/workflows/config.json

```diff
@@ -2,6 +2,6 @@
   "deployment" : {
     "repository_owner" : "elementor",
     "environment" : "prod",
-    "permitted" : "KingYes,shilo-ey,matipojo,TzviRabinovitch,blackcolt,davseve,louiswol94,mykytamurzin,hein-obox,Barcov,ArmanPurtoian,dmaziar,ElementorBot"
+    "permitted" : "KingYes,shilo-ey,matipojo,mark-elementor,manorhazaz,omerisra6,rotemee,styleShit,TzviRabinovitch,blackcolt,davseve,louiswol94,mykytamurzin,hein-obox,Barcov,ArmanPurtoian,dmaziar,ElementorBot"
```

Comment:
@blackcolt Just a question, are these usernames not case-sensitive?
eg StyleShit, ManorHazaz
Or will they work like this?